### PR TITLE
[release] merging crate-release-update into crate

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @GuillaumeGomez @EPashkin

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,4 +16,4 @@ script:
   - echo 'paths = ["."]' > .cargo/config
   - git clone -q --depth 50 -b pending https://github.com/rust-gnome/examples _examples
```
Comment (EPashkin) [inline .travis.yml]: This IMHO need be `github.com/gtk-rs/examples` and on master branch,
but seem it need be done after merge or at minimum after merge "crate-release-update into master"

Comment (GuillaumeGomez) [reply .travis.yml]: Oh wow. That's quite the old URL!

Comment (GuillaumeGomez) [reply .travis.yml]: BTW, we need to figure out what's happening with cairo before moving forward on the release (for live time info, I recommend coming on IRC ;) ).

Comment (EPashkin) [reply .travis.yml]: @GuillaumeGomez Which IRC? You seems not on #gtk-rs on irc.mozilla.org

Comment (GuillaumeGomez) [reply .travis.yml]: Answered on the IRC. :)

**Pull request closed without merging**